sdk/keyvault/Azure.Security.KeyVault.Secrets/ChangeLog.md

```diff
@@ -1,5 +1,19 @@
 # Release History
 
+## 4.0.0-preview.6
+
+### Breaking changes
+
+- `Secret` has been renamed to `KeyVaultSecret` to avoid ambiquity with other libraries and to yield better search results.
+- `VaultUri` properties and `vaultUri` parameters have been renamed to `VaultEndpoint` and `vaultEndpoint` respectively.
+- `SecretClient.GetSecrets` and `SecretClient.GetSecretVersions` have been renamed to `SecretClient.GetPropertiesOfSecrets` and `SecretClient.GetPropertiesOfSecretVersions` respectively.
+- `SecretClient.RestoreSecret` has been renamed to `SecretClient.RestoreSecretBackup` to better associate it with `SecretClient.BackupSecret`.
+
```

Comment:
Expires => ExpiredOn?

Reply:
Good catch. Must've forgot that one for Secrets's README.

Reply:
...though I do wonder, is this *too* much detail, or just right for a CHANGELOG? Obviously, we'll distill this for the general client CHANGELOG, but what about the nupkg release notes?